src/classes/EscalationTrigger_Test.cls

```diff
@@ -186,4 +186,58 @@ public class EscalationTrigger_Test {
 		System.assertEquals(null, testEscalation.rh_escal__PrimaryCase__c, 'The primary case of escalation should be null.');
 		System.assertEquals(null, testEscalation.rh_escal__Account__c, 'The account is not set correctly.');
 	}
+
+	////////////////////////////////////////////////////////////////////////////////////
+	//                   createEscalationCaseRecordOnEscalationInsert                 //
+	////////////////////////////////////////////////////////////////////////////////////
+
+	static testMethod void createEscalationCaseRecord_primaryCaseOnEscalationNotNull_valid() {
+		Account testAccount = TestUtils.getAccount();
+		insert testAccount;
+
+		Case testCase = TestUtils.getCase(testAccount);
+		insert testCase;
+
+		rh_escal__Escalation__c testEscalation = TestUtils.getEscalation();
+		TestUtils.setPrimaryCaseOnEscalation(testEscalation, testCase);
+
+		Test.startTest();
+
+		insert testEscalation;
+
+		Test.stopTest();
+
+		testEscalation = TestUtils.fetchEscalation(testEscalation);
+
+		rh_escal__EscalationCase__c escalCase = TestUtils.getEscalationCase(testEscalation, testCase);
+
+		System.assertEquals(testCase.Id, testEscalation.rh_escal__PrimaryCase__c, 'The primary case is not set correctly.');
+		System.assertEquals(testAccount.Id, testEscalation.rh_escal__Account__c, 'The account is not set correctly.');
+		System.assertNotEquals(null, escalCase, 'Did not get the expected escal case record.');
```

Comment:
Also add some asserts to make sure that all three fields you set in the trigger are set in the `escalCase` object

```java
System.assertEquals(testCase.Id, escalCase.rh_escal__Case__c, 'The case id was not set correctly');
System.assert(escalCase.rh_escal__Primary__c, 'This should be the primary case');
System.assertEquals(testEscalation.Id, escalCase.rh_escal__Escalation__c, 'The escalation id was not set correctly');